src/Components/CollectionsHubRails/ArtistSeriesRail/__tests__/ArtistSeriesEntity.test.tsx

```diff
@@ -13,34 +13,86 @@ describe("ArtistSeriesEntity", () => {
     }
   })
 
-  it("Renders expected fields", () => {
+  it("showing the correct text, price guidance, amount of hits and image", () => {
     const component = mount(<ArtistSeriesEntity {...props} />)
-
-    expect(component.text()).not.toMatch("Jasper Johns:")
-    expect(component.text()).toMatch("Flag")
+    expect(component.text()).toMatch("Flags unique collections")
     expect(component.text()).toMatch("From $1,000")
     expect(component.find(ArtworkImage).length).toBe(3)
     const artworkImage = component
       .find(ArtworkImage)
       .at(0)
       .getElement().props
-
     expect(artworkImage.src).toBe(
       "https://d32dm0rphc51dk.cloudfront.net/4izTOpDv-ew-g1RFXeREcQ/small.jpg"
     )
-    expect(artworkImage.alt).toBe("Jasper Johns, Flag")
+  })
+
+  it("NOT rendering any artist name or the title of the artwork ", () => {
+    const component = mount(<ArtistSeriesEntity {...props} />)
+    expect(component.text()).not.toMatch("A great flag from Jasper")
+    expect(component.text()).not.toMatch("Jasper Johns")
+    expect(component.text()).not.toMatch("Back to 2046")
+    expect(component.text()).not.toMatch("Andy Warhol")
+    expect(component.text()).not.toMatch("An Apple")
+  })
+
+  it("uses small image width when there are more than 2 hits", () => {
+    const component = mount(<ArtistSeriesEntity {...props} />)
+    const artworkImage = component
```

Comment:
This is a very subtle change, but assertions should be written in order of specificity. The more general assertions should come first, and the more specific ones should come last. In this example, line 45 is the most general assertion - it should come before line 41. 

This is because tests fail on the first assertion that fails. It's helpful when looking at a failing test if the assertions are written in order of specificity, because it helps you narrow down exactly where your assertions went wrong. The way this code is currently written, if there were zero `ArtworkImage`'s, line 43 would fail because there would be no element `.at(0)`. This _might_ give us an error message that tells us that there are no `ArtworkImage` elements, but it also might not. The assertion in line 45 would _definitely_ give us that error message - so it should come first.

Let me know if this doesn't make sense!